doc/contributing/releases.md

```diff
@@ -1207,6 +1207,18 @@ Notify the `@nodejs/npm` team in the release proposal PR to inform them of the
 upcoming release. `npm` maintains a list of [supported versions](https://github.com/npm/cli/blob/latest/lib/utils/unsupported.js#L3)
 that will need updating to include the new major release.
 
+To keep the branch in sync until the release date, it can be as simple as
+doing the following:
+
+```bash
+git checkout vN.x
+git reset --hard upstream/main
```

Comment:
This might not be desirable if there are any changes on `main` with the appropriate `dont-land-on-*` label for the major being prepared, e.g. [`dont-land-on-v21.x`](https://github.com/nodejs/node/labels/dont-land-on-v21.x).

Reply:
That's true. What would be an easy approach then? It's unlikely to have a PR with the label `dont-land-on-vN.x` when preparing a major release, looks like adding a comment to manually check it before running the command might be simpler.

Reply:
A comment to check would be sufficient for me.

Reply:
Maybe open an issue in nodejs/Release and discuss it there? I'll go ahead and add a comment